test/integration/targets/inventory_aws_conformance/runme.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+set -ex
+
+TARGET=$(pwd)
+
+# set the output dir
+if [ -z "${OUTPUT_DIR+null}" ]; then
+    export OUTPUT_DIR=$TARGET
+fi
```

Comment:
Why have an extra var for TARGET, or logic for setting OUTPUT_DIR when it's already set?